OpenFoodFacts.xcodeproj/project.pbxproj

```diff
@@ -1859,7 +1859,7 @@
 					};
 					95C2650D1E96D55B004212EC = {
 						CreatedOnToolsVersion = 8.3;
-						DevelopmentTeam = ZC9CYWD334;
+						DevelopmentTeam = N4FDL334LZ;
```

Comment:
Can you avoid tweak  Dev Teams in the PRs ? :-)

Reply:
sure my bad , i changed them so that i could test it on my phone, will make sure to not repeat the mistake.

Reply:
Please revert the change, ammend to your commit and force push so it will be like you never did it ;)

Reply:
ok sure